docs/configuration/optional-settings.md

```diff
@@ -86,8 +86,17 @@ Note that `USE_TLS`/`USE_SSL` are mutually exclusive, so only set one of those s
 
 ---
 
+## PEERINGDB_API_KEY
+
+PeeringDB API Key used to authenticate against PeeringDB allowing Peering Manager to sync
+Email contacts.
+
+---
+
 ## PEERINGDB_USERNAME / PEERINGDB_PASSWORD
 
+WARNING: DEPRECIATED, PLEASE USE `PEERINGDB_API_KEY` instead.
```

Comment:
Would use "deprecated" instead of "depreciated", since the latter probably is unknown by foreign speakers.
```suggestion
WARNING: DEPRECATED, PLEASE USE `PEERINGDB_API_KEY` INSTEAD.
```